src/components/Sidebar/CollectionList.svelte

```
@@ -4,7 +4,7 @@
 	import { userIconCollections } from '../../stores/user-icon-collection.store';
 	import { selectedCollection } from '../../stores/selected-collection.store';
 	import { Accordion } from '@skeletonlabs/skeleton-svelte';
-	import * as _selectedIcon from '../../models/SelectedIcon';
+	import * as _selectedIcon from '../../models/CustomizableIcon';
```

ESLint

[error] '_selectedIcon' is defined but never used.

(@typescript-eslint/no-unused-vars)